net/ser2net/files/etc/config/ser2net

```diff
@@ -0,0 +1,7 @@
+config 'proxy'
+        option 'tcpport' '8000'
+        option 'state' 'raw'
+        option 'timeout' '30'
```

Comment:
Please add proper default handling when create config file in case `timeout` option is not given in UCI.